CHANGELOG.md

```diff
@@ -4,24 +4,28 @@ The format is described in [Contributing notes](CONTRIBUTING.md#changelog-entry-
 
 ## master (unreleased)
 
-### Added
-
-* todo
-
-### Changed
-
-* todo
+## 0.1.0 (2019-06-28)
 
-### Bug fixes
-
-* todo
-
-### Removed
+### Added
 
-* todo
+* Base config with community rules and some basic override rules. ([@r.dubrovsky][], [@aleks][])
+* Config based on rubocop version 0.72.0 ([@r.dubrovsky][])
+* Enable `rubocop-spec` cops by default ([@r.dubrovsky][])
+* Enable `Bundler/DuplicatedGem` cop by default ([@r.dubrovsky][])
+* Enable `Bundler/OrderedGems` cop with allowing ordering by groups ([@r.dubrovsky][])
 
-## 0.1.0 (date here e.g 2019-05-25)
 
-### Added
+### Changed
 
-* Base config with community rules and some basic override rules.
+* The limit for size of line was changed to 120 symbols ([@r.dubrovsky][])
+* Disable `Metrics/BlockLength` cop for rspec files ([@r.dubrovsky][])
+* Exclude rubocop checking for some config diractories ([@r.dubrovsky][])
+* Enable prafering double quotes for string literals ([@r.dubrovsky][])
+* Do not add spaces beetwen hash literal braces (cop `Layout/SpaceInsideHashLiteralBraces`) ([@r.dubrovsky][])
+* Prefer normal style for `Layout/IndentationConsistency` cop for Rails apps too ([@r.dubrovsky][])
+* Change style to `variable` for `Layout/EndAlignment` cop ([@r.dubrovsky][])
+* Change style to `with_fixed_indentation` with indentation width 2 for `Layout/AlignParameter` cop ([@r.dubrovsky][])
+* Always ingone hash aligning for key word arguments (cop: `Layout/AlignHash`) ([@r.dubrovsky][])
+
+[@r.dubrovsky]: https://github.com/roman-dubrovsky
+[@aleks]: https://github.com/AleksSenkou
```

Comment:
@roman-dubrovsky thoughts about marking those who helped with code review?

Reply:
@AleksSenkou have you meant doing it for the current changes or in general?

Reply:
@roman-dubrovsky not sure, just wondering

Reply:
@AleksSenkou no thoughts. In general, I don't think that it makes sense. Usually, we'll make the big impact of some feature (e.g. describe and implement some feature). But if the feature creator feels that the reviewer made a great impact he can add this person to the contributes as well. Does it make sense?

Reply:
@roman-dubrovsky yeah, sounds reasonable